### ISSUE TITLE
Login Request won't reach server

### ISSUE THREAD

**Issue by lRaikeNl** [vthibault/roBrowser#134]
Hello everyone, sorry for writing my post here instead of the forum. I can't register to the forum and no one is replying in the IRC Chat to help me setup roBrowser.

I've done everything like described on the getting started tutorial, now I'm stuck at the part where I Login and connect to my server, but no Login packet is reaching my server.

the console doesn't give any error except the BGM play thingy.

Please help...

**Comment by Akkarinage** [comment]
Open the js console on your browser (usually ctrl + shift+ i) and see if you get any errors there.

**Comment by lRaikeNl** [comment]
Nope, no errors. Do I need something like websocket? I don't use any proxy or socket thingy.
